packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/normal-dialing.js

```diff
@@ -79,13 +80,13 @@ describe('samples', () => {
         browserSpock.waitUntil(() => (browserSpock.getText('#screenshare-tracks') === 'STOPPED'), 10000, 'Timed-out waiting for screenshare to stop');
       });
 
-      it('starts screensharing remote', () => {
+      flaky(it, process.env.SKIP_FLAKY_TESTS)('starts screensharing remote', () => {
         browserMccoy.click('button[title="share screen"]');
 
         browserMccoy.waitUntil(() => (browserMccoy.getText('#screenshare-tracks') === 'SHARING'), 10000, 'Timed-out waiting for screenshare to start');
       });
 
-      it('stops screensharing remote', () => {
+      flaky(it, process.env.SKIP_FLAKY_TESTS)('stops screensharing remote', () => {
```

Comment:
I’d only mark this one as flaky, as this is the test that is consistently failing.
The screen does share, however it doesn’t close, and we should at least test that acreensharing works on chrome